Sources/Remote Logging/Crash Logging/CrashLogging.swift

```diff
@@ -168,54 +171,12 @@ public extension CrashLogging {
     }
 
     /// Sends an `Event` to Sentry and triggers a callback on completion
-    func logErrorImmediately(_ error: Error, userInfo: [String: Any]? = nil, level: SentryLevel = .error, callback: @escaping (Result<Bool, Error>) -> Void) throws {
+    func logErrorImmediately(_ error: Error, userInfo: [String: Any]? = nil, level: SentryLevel = .error, callback: @escaping () -> Void) throws {
         try logErrorsImmediately([error], userInfo: userInfo, level: level, callback: callback)
     }
 
-    func logErrorsImmediately(_ errors: [Error], userInfo: [String: Any]? = nil, level: SentryLevel = .error, callback: @escaping (Result<Bool, Error>) -> Void) throws {
-
-        var serializer = SentryEventSerializer(dsn: dataProvider.sentryDSN)
-
-        errors.forEach { error in
-            let event = Event.from(
-                error: error as NSError,
-                level: level,
-                user: dataProvider.currentUser?.sentryUser,
-                extra: userInfo ?? (error as NSError).userInfo
-            )
-
-            event.threads = currentThreads()
-
-            serializer.add(event: event)
-        }
-
-        guard let requestBody = try? serializer.serialize() else {
-            TracksLogError("⛔️ Unable to send errors to Sentry – error could not be serialized. Attempting to schedule delivery for another time.")
-            errors.forEach {
-                SentrySDK.capture(error: $0)
-            }
-            return
-        }
-
-        let dsn = try SentryDsn(string: dataProvider.sentryDSN)
-        guard let authString = dsn.getAuthString() else {
-            throw Errors.unableToConstructAuthStringError
-        }
-
-        var request = URLRequest(url: dsn.getEnvelopeEndpoint())
-        request.httpMethod = "POST"
-        request.httpBody = requestBody
-        request.addValue(authString, forHTTPHeaderField: "X-Sentry-Auth")
-
-        URLSession.shared.dataTask(with: request) { (responseData, urlResponse, error) in
-            if let error = error {
-                callback(.failure(error))
-                return
-            }
-
-            let didSucceed = 200...299 ~= (urlResponse as! HTTPURLResponse).statusCode
-            callback(.success(didSucceed))
-        }.resume()
+    func logErrorsImmediately(_ errors: [Error], userInfo: [String: Any]? = nil, level: SentryLevel = .error, callback: @escaping () -> Void) throws {
```

Comment:
I had a quick look, it appears we can't get the result of `flush` call, is that correct? I think it would be good to at least have a "success or not" result, if we can't get a concrete error. But I guess there isn't anything we can do if Sentry doesn't expose such API.